homeassistant/components/kraken/const.py

```diff
@@ -0,0 +1,141 @@
+"""Constants for the kraken integration."""
+
+DATA_UPDATED = "kraken_data_updated"
+
+DEFAULT_SCAN_INTERVAL = 5
+
+DOMAIN = "kraken"
+
+TRADEABLE_ASSET_PAIRS = [
```

Comment:
Knowing Kraken... This will change pretty often! Can't we fetch it some from somewhere?

Reply:
Yes we can.
What would happen with sensors for asset pairs which will get removed?
The would just stop showing data but cannot be removed without removing the whole integration, am I right?

Reply:
When I look how coinbase is doing it, it loads the available currencies on setup and logs an error when there is a configured currency not available.

1. This still means that during the uptime of hass the currency could become unavailable and would lead to an uncaught error, correct?
2. As I understand the design decision was that it should not be possible anymore to restrict collected data in the configuration. Rather get all available data, create all available entities and let the user decide in the frontend what to do. Correct?

Reply:
IMHO we should mark the sensor as unavailable at that point.